crates/thetawave_interface/src/options/input.rs

```diff
@@ -0,0 +1,79 @@
+use bevy_ecs::{component::Component, system::Resource};
+use bevy_input::{gamepad::GamepadButtonType, keyboard::KeyCode, mouse::MouseButton};
+use bevy_reflect::Reflect;
+use leafwing_input_manager::{prelude::InputMap, Actionlike};
+use serde::Deserialize;
+
+#[derive(Component)]
+pub struct MenuExplorer;
+
+#[derive(Actionlike, PartialEq, Eq, Clone, Copy, Hash, Debug, Reflect, Deserialize)]
+pub enum MenuAction {
+    Confirm,
+    JoinKeyboard,
+    ChangeCharacterKeyboard,
+    JoinGamepad,
+    ChangeCharacterGamepad,
+    Back,
+    Reset,
+    ExitPauseMenu,
+    PauseGame,
+    ToggleTutorial,
+}
+
+#[derive(Actionlike, PartialEq, Eq, Clone, Copy, Hash, Debug, Reflect, Deserialize)]
+pub enum PlayerAction {
+    MoveUp,
+    MoveDown,
+    MoveLeft,
+    MoveRight,
+    BasicAttack,
+    SpecialAttack,
+}
+
+#[derive(Deserialize)]
+pub struct InputBindings {
```

Comment:
```suggestion
struct InputBindings {
```

Reply:
This needs to remain public:
https://github.com/thetawavegame/thetawave/pull/108/files#diff-bbff6d4c12e9078362fb1f4999f7ed294baa96c19c6fb33345937573334c1527R47